mobly/controller_manager.py

```diff
@@ -125,12 +125,15 @@ def register_controller(self, module, required=True, min_number=1):
       original_config = self.controller_configs[module_config_name]
       controller_config = copy.deepcopy(original_config)
       objects = module.create(controller_config)
-    except Exception:
-      logging.exception(
-          'Failed to initialize objects for controller %s, abort!',
-          module_config_name,
+    except Exception as e:
+      msg = (
+          f'Failed to initialize objects for controller "{module_config_name}". '
+          'Check if the device is connected and the config is correct.'
```

Comment:
a controller can be anything, so we intentionally didn't say "device" in the error message.
and we should not have Android specific logic in the generic controller manager module.

I'm rejecting this PR